content/installation/python/PythonConfig.md

```diff
@@ -90,17 +90,17 @@ Define the following properties to set Syslog values. If the properties aren't d
 The following properties are used by the Contrast Service.
 
 * **service**: 
-  * **enable**: Set to `false` to disallow the service to be started, and effectively disable the agent, if read by the service. If the agent reads this property, it disallows service auto-start.
-  * **socket**: If this property is defined, the service is listening on a Unix socket at the defined path. <br> Example: /tmp/service.sock
-  * **host**: Set the the hostname or IP address of the Contrast service to which the Contrast agent should report. **Required.** <br> Example: `localhost`
-  * **port**: Set the the port of the Contrast service to which the Contrast agent should report. **Required.** <br> Example: `30555`
+  * **enable**: Set to `false` to prevent the agent from automatically starting the service. This is necessary when connecting to a standalone instance of the service. Defaults to `true`.
+  * **socket**: If this property is defined, and `enable` is set to `true`, the service will be started on a Unix socket at the defined path. Not compatible with the use of `host` and/or `port`. <br> Example: /tmp/service.sock
+  * **host**: If `enable` is `true`, bind the service to this hostname. Otherwise, connect to an external service at this hostname. Defaults to `localhost`. Not compatible with `socket`. <br> Example: `localhost`
+  * **port**: If `enable` is `true`, bind the service to this port. Otherwise, connect to an external service on this port. Defaults to `30555`. Not compatible with `socket`. <br> Example: `30555`
     
```

Comment:
@drdavella What was the original PR for these changes to property descriptions?

Reply:
It looks like it was #508.